cura/LayerPolygon.py

```diff
@@ -186,6 +187,11 @@ def extruder(self):
     def types(self):
         return self._types
 
+    @property
+    def lineLengths(self):
```

Comment:
How are you gonna treat the last (or initial) travel move in this regard?

Reply:
I am sorting it out w.r.t.  to visible indexs here: https://github.com/Ultimaker/Cura/blob/efda0aaba4329948485021bbbc4d2e365a5b63c4/plugins/SimulationView/SimulationView.py#L532

Reply:
Correction: All travel moves are calculated using 2 consecutive points, so if number of vertices in a layer is 701, the lengths covered will be 700.